testdata/amazon-eks-nodegroup.yaml

```diff
@@ -74,7 +74,7 @@ Parameters:
 
   NodeImageIdSSMParam:
     Type: "AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>"
-    Default: /aws/service/eks/optimized-ami/1.22/amazon-linux-2/recommended/image_id
+    Default: /aws/service/eks/optimized-ami/1.24/amazon-linux-2/recommended/image_id
```

Comment:
Nit: Consider adding a constant for `1.24`, I believe we have few places where we refer to K8s versions..

Reply:
Good call, yeah this should be a parameter